etna/core/mixins.py

```diff
@@ -89,6 +90,49 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    @staticmethod
+    def _update_nested_dict_with_flat_dict(params_dict: dict, flat_dict: dict):
+        """Change flat specification into dict of nested params.
+
+        Parameters
+        ----------
+        **params_dict: dict
+            dict with nested parameters structure, e.g ``{"model": {"learning_rate": value1}}``
+        **flat_dict: dict
+            dict with flat paratemers structure, e.g. ``{"model.depth": value2}``
+
+        Returns
```

Comment:
As I can see, this method returns nothing, this section should be removed. Probably, you can rewrite description of method to fill it with this information.